daemon/src/projection.rs

```diff
@@ -729,6 +729,28 @@ impl Cfd {
 
         Some(url)
     }
+
+    fn is_open(&self) -> bool {
+        let in_open_state = vec![
+            CfdState::PendingOpen,
+            CfdState::Open,
+            CfdState::PendingCommit,
+            CfdState::OpenCommitted,
+            CfdState::IncomingRolloverProposal,
+            CfdState::IncomingSettlementProposal,
+            CfdState::OutgoingRolloverProposal,
+            CfdState::OutgoingSettlementProposal,
+            CfdState::RolloverSetup,
+        ]
+        .contains(&self.state);
+
+        let not_expired = match self.expiry_timestamp {
+            None => true,
+            Some(expiry_timestamp) => expiry_timestamp > OffsetDateTime::now_utc(),
+        };
```

Comment:
🤔 I suppose we've had this logic elsewhere before, but I'm not sure I agree.

The (confusingly named) `expiry_timestamp` corresponds to the time at which the oracle is expected to attest to an event that would allow non-collaborative settlement. Even if we have reached this time:

- The oracle could fail to attest in time.
- The commit transaction could take a while to be confirmed.

I don't see why we need to use this value to reason about the openness of the CFD

Reply:
Yea, we use the same logic in other places outside of this project :)

Now that I think about it a bit more, for metrics this is actually not that important. It is only important for out automation. 
We should derive `is_closed` only from the states... 

Thanks for pointing this out 🙏